website/docs/docs/collaborate/column-level-lineage.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Column level lineage"
+description: "dbt Explorer provides recommendations that you can take to improve the quality of your dbt project."
+---
+
+dbt Explorer now offers column level lineage (CLL) for the resources in your dbt project. Analytics engineers can quickly and easily gain insight into the provenance of their data products at a more granular level. For each column in a resource (model, source, or snapshot) in a dbt project, Explorer provides end-to-end lineage for the data in that column given how it's used.
+
+Column level lineage is available to dbt Cloud Enterprise accounts that can use Explorer. It’s also available through the Discovery API.
+
+:::tip Beta
+Column-level lineage is now available in beta. Check it out! We'd love to [know what you think](https://docs.google.com/forms/d/e/1FAIpQLSdpCbVkGY9QwfExFonpWE4DTOKi3fQxBGLD0wwKYpkMjgcE7g/viewform)!
+:::
+
+## Access the column level lineage
+
+There is no additional setup required for column level lineage if your account is on an Enterprise plan that can use Explorer. You can access the column level lineage by expanding the column card in the **Columns** tab of an Explorer [resource details page](/docs/collaborate/explore-projects#view-resource-details) for a model, source, or snapshot.
+
+dbt updates the lineage after each run that's executed in the production environment. You must make sure that `docs generate` is running within at least one job in the environment. Refer to [Generating metadata](/docs/collaborate/explore-projects#generate-metadata) for more details.
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/example-cll.png" title="Example of the Columns tab and where to expand for the CLL"/>
+
+<LoomVideo id='278c948ba387457884cc6b9545793685' />
+
+## Column level lineage use cases {#use-cases}
```

Comment:
updated heading title so it shows up higher (ie "weighted" more) in our search list as well as keeping the URL tidy. here's a comparison example: 

![Screenshot 2024-01-19 at 8 51 40 AM](https://github.com/dbt-labs/docs.getdbt.com/assets/107218380/31f31ad7-39f6-4eb4-a773-95064032312c)
![Screenshot 2024-01-19 at 8 52 27 AM](https://github.com/dbt-labs/docs.getdbt.com/assets/107218380/9d39799b-b9f5-442e-841b-0427f3245cf7)